projects/demo/src/modules/components/axes/index.ts

```diff
@@ -1,36 +1,40 @@
+import {NgForOf, NgIf} from '@angular/common';
 import {Component} from '@angular/core';
 import {changeDetection} from '@demo/emulate/change-detection';
+import {
+    TuiComponentPipe,
+    TuiExamplePipe,
+    TuiKebabPipe,
+    TuiSetupComponent,
+} from '@demo/utils';
 import type {TuiLineHandler, TuiLineType} from '@taiga-ui/addon-charts';
-import {TUI_ALWAYS_DASHED, TUI_ALWAYS_SOLID} from '@taiga-ui/addon-charts';
-import type {TuiDocExample} from '@taiga-ui/addon-doc';
+import {
+    TUI_ALWAYS_DASHED,
+    TUI_ALWAYS_SOLID,
+    TuiAxesComponent,
+} from '@taiga-ui/addon-charts';
+import {TuiAddonDocModule} from '@taiga-ui/addon-doc';
+import {TuiNotificationModule} from '@taiga-ui/core';
 
 @Component({
-    selector: 'example-tui-axes',
-    templateUrl: './axes.template.html',
-    styleUrls: ['./axes.style.less'],
+    standalone: true,
+    imports: [
+        TuiAddonDocModule,
+        TuiAxesComponent,
+        TuiNotificationModule,
+        TuiSetupComponent,
```

Comment:
Like in every other PRs like that, just import `TuiDemoModule` please for brevity.